_data/sites.json

```diff
@@ -2886,21 +2886,19 @@
 
     {
         "name": "EdX",
-        "url": "https://www.edx.org/student-faq",
-        "difficulty": "impossible",
-        "notes": "There's no need to delete your account. An old, unused edX account with no course completions associated with it will disappear.",
-        "notes_tr": "Hesabınızı silmenize gerek yoktur. Kurs tamamlamaları olmayan ve eski, kullanılmamış bir edX hesabı kendiliğinden kaybolacaktır.",
-        "notes_fr": "Il ne faut pas supprimer votre compte. Un compte edX qui n'est pas utilisé et qui n'a pas d'achèvement de cours disparaîtra.",
-        "notes_de": "Es gibt keine Notwendigkeit deinen Account zu löschen. Ein alter, unbenutzter edX Account ohne fertiggestellte Kurse wird automatisch gelöscht.",
-        "notes_it": "Non è necessario cancellare il tuo account. Un vecchio, inusato edX account scomparirà da solo",
-        "notes_pt_br": "Não há porque deletar sua conta. Uma conta antiga, e não usada sem cursos completos irá desaparecer.",
-        "notes_cat": "No té perqué esborrar el seu compte. Un compte antic sens ús ni cursos completats serà eliminat.",
-        "notes_es": "«No hay necesidad de borrar la cuenta. Una cuenta antigua no usada y sin cursos completos será eliminado.»",
+        "url": "https://account.edx.org/#delete-account",
+        "difficulty": "medium",
+        "notes": "Go to the account deletion page and select Delete Account. If you have logged in through a social account, you'll first have to unlink them, reset your password then delete the account.",
+        "notes_tr": "Hesap silme sayfasına gidin ve Hesabı Sil'i seçin. Bir sosyal hesap üzerinden oturum açtıysanız, önce bunların bağlantısını kaldırmanız, şifrenizi sıfırlamanız ve ardından hesabı silmeniz gerekir.",
+        "notes_fr": "Accédez à la page de suppression de compte et sélectionnez Supprimer le compte. Si vous vous êtes connecté via un compte social, vous devrez d'abord les dissocier, réinitialiser votre mot de passe puis supprimer le compte.",
+        "notes_de": "Gehen Sie zur Seite zum Löschen des Kontos und wählen Sie Konto löschen. Wenn Sie sich über ein soziales Konto angemeldet haben, müssen Sie zuerst die Verknüpfung aufheben, Ihr Passwort zurücksetzen und dann das Konto löschen.",
+        "notes_it": "Vai alla pagina di eliminazione dell'account e seleziona Elimina account. Se hai effettuato l'accesso tramite un account social, dovrai prima scollegarli, reimpostare la password, quindi eliminare l'account.",
```

Comment:
Do you know all these languages or did you use machine translation?
It's alright to delete translations when the notes are updated, we prefer doing that than doing machine translation

Reply:
Ah no I used machine translation sorry.

Reply:
No worries, we can just wipe them.
```suggestion
```